ash/src/vk/prelude.rs

```diff
@@ -1,10 +1,12 @@
+use super::super::vk;
```

Comment:
This is still up for debate, see https://github.com/MaikKlein/ash/pull/557#discussion_r787534974.

Reply:
I favor going through the root in ash's case, since it's fairly concise, makes impl code read like usage code, and makes the docs easier.

Reply:
Fair enough, my only objection would be to make this `mod` (or rather it's parent `mod vk`) self-supporting, ie. oblivious to where it's located in the tree. No strong feelings though and I prefer your suggestion if this is considered more of a "copy paste this is how you can use these types" example.